### ISSUE TITLE
Non git package for the AUR

### ISSUE THREAD

**Issue by 4censord** [dyc3/steamguard-cli#172]
> Arch-based systems can install from the AUR:
>  For [steamguard-cli-git](https://aur.archlinux.org/packages/steamguard-cli-git/)
>  Non-git release is not officially provided. Please open an issue if you would like to help set that up.

What would be required to set this up?

**Comment by dyc3** [comment]
It turns out that someone is already maintaining a non-git package on the aur: https://aur.archlinux.org/packages/steamguard-cli

So I guess the next steps are to reach out to them to see if we can make it official.

**Comment by xenrox** [comment]
Hey,

I have made you a Co-Maintainer of the aur package. Is there anything else that you want me to do?

**Comment by dyc3** [comment]
Thanks @xenrox, I'll go ahead and update the readme. Do you have the packaging automated? I'd like to integrate the packaging into my release workflow.

**Comment by xenrox** [comment]
No, right now I check the release notes, build the package in a chroot and if everything works, I push to the aur.

I personally don't know much about GitHub workflows, so I cannot help much there,

**Comment by dyc3** [comment]
Alright, I'll leave this issue open for now to remind me to work on automating it later.

**Comment by C0rn3j** [comment]
You can at the very least easily automate bumping checksums and making sure it builds in a clean chroot.

https://gitlab.com/C0rn3j/configs/-/blob/34dd12a72e10e911a50baecfef4e4672e5de5e74/roles/tools/files/corn-makepkg.sh#L9

You can probably also automate version bumps through nvchecker.
